### PR TITLE
Make driver camera automatically appear in Shuffleboard

### PR DESCRIPTION
**Pull request by KangarooKoala**
Move driver vision camera to DrivebaseSubsystem
Add driver camera to drivebase tab

### CODE REVIEW

**Comment by KangarooKoala** [comment]
I don't have a device to run Shuffleboard on, so someone should run it to check that it works right.

**Comment by TAKBS2412** [comment]
Will review later tonight

**Review by TAKBS2412 — COMMENTED** [review]
I tested on my machine and there's some really interesting behavior going on. When I open shuffleboard first, then run the simulator, the stream shows up as expected (currently a black screen since I don't have a USB camera plugged in at the moment). However, if I run the simulator then open shuffleboard, the camera widget does not show up at all. This suggests to me that there is a race condition somewhere, either in our software or WPILib itself.

I would be interested in hearing if others can reproduce this behavior.

**Comment by SumedhP** [comment]
Haven't been able to test, but looks good. Going forward with the addition of a codriver monitor, we may want to look into using the web interface to display both this camera and the LL as well if shuffleboard may not open reliably

File: src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java [hunk 1]
```diff
@@ -151,6 +155,16 @@ public DrivebaseSubsystem() {
...
         moduleMaxVelocityMetersPerSec = MODULE_MAX_VELOCITY_METERS_PER_SEC;
 
+        if (DRIVER_VIS_ENABLED) {
+            UsbCamera driverCamera = new UsbCamera("Driver Vision Front", Hardware.FRONT_CAM);
+            driverCamera.setResolution(160, 90);
+            CameraServer.addCamera(driverCamera);
+            CameraServer.startAutomaticCapture();
+            tab.add("Driver camera", driverCamera)
+                    .withPosition(5, 0)
+                    .withSize(2, 2);
```
Comment (SumedhP) [inline src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java:165]: Small nit: the camera feed is quite small at 2x2 widget size and is far more readable at 4x4, so increasing the default size to that would be nice

**Comment by TAKBS2412** [comment]
> Haven't been able to test, but looks good. Going forward with the addition of a codriver monitor, we may want to look into using the web interface to display both this camera and the LL as well if shuffleboard may not open reliably

Yeah I would have the highest confidence in just using the web interface at this point. Having to open shuffleboard beforehand every time seems like a hassle.

**Comment by KangarooKoala** [comment]
What kind of web interface does the monitor use?

**Comment by KangarooKoala** [comment]
Decided we'd use the web interface instead of Shuffleboard, so this is unnecessary.

**Pull request closed without merging**